packages/transport-grpc/src/send-get-block-header.test.js

```diff
@@ -1,34 +1,19 @@
 import {AccessAPI} from "@onflow/protobuf"
 import {sendGetBlockHeader} from "./send-get-block-header.js"
-import {build} from "../../sdk/src/build/build.js"
-import {getBlockHeader} from "../../sdk/src/build/build-get-block-header.js"
-import {atBlockId} from "../../sdk/src/build/build-at-block-id.js"
-import {atBlockHeight} from "../../sdk/src/build/build-at-block-height.js"
-import {resolve} from "../../sdk/src/resolve/resolve.js"
-import {response as responseADT} from "../../sdk/src/response/response.js"
 import {Buffer} from "@onflow/rlp"
-
-const jsonToUInt8Array = json => {
-  var str = JSON.stringify(json, null, 0)
-  var ret = new Uint8Array(str.length)
-  for (var i = 0; i < str.length; i++) {
-    ret[i] = str.charCodeAt(i)
-  }
-  return ret
-}
+import {
+  atBlockHeight,
+  atBlockId,
+  build,
+  getBlockHeader,
+  resolve,
+  response as responseADT,
+} from "@onflow/sdk"
```

Comment:
I believe this creates a circular dep between `@onflow/sdk` & `@onflow/transport-grpc` which we should avoid 🤔

Reply:
Also - I don't see `@onflow/sdk` in this packages dependencies which may cause issues

Reply:
Okay, added it as a dev dependency, and we are no longer including the src/* files in the npm package.